### ISSUE TITLE
[jdbc][jdbc parameter] Muilt Table Sync Use same jdbc connection properties 

### ISSUE THREAD

**Issue by itcrazy0918** [apache/seatunnel#2241]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

The database connection information in the JDBC source table & sink table needs to be written repeatedly, and cannot be configured only once

suggest Solution: extract it into env module for unified definition or external parameter reference

### Usage Scenario

When synchronizing multiple tables, avoid repeatedly writing database connection information, and configure it only once 
eg :

env {
    spark.app.name = "sync_base_tables"
    spark.executor.instances = 1
    spark.executor.cores = 1
    spark.executor.memory = "1g"
}

jdbcconfig {
    driver = "com.mysql.jdbc.Driver"
    url = "jdbc:mysql://127.0.0.1:3306/testdb"
    user = "root"
    password = "123456"
}

xxxconfig{
    ......
}

source {
    jdbc {
         ${jdbcconfig}
        table = "location"
        result_table_name = "location"
        jdbc.fetchsize = 500
        jdbc.batchsize = 500
    }
    jdbc {
        ${jdbcconfig}
        table = "subsystem"
        result_table_name = "subsystem"
        jdbc.fetchsize = 500
        jdbc.batchsize = 500
    }
    jdbc {
        ${jdbcconfig}
        table = "me_alarm_type"
        result_table_name = "tablec"
        jdbc.fetchsize = 500
        jdbc.batchsize = 500
    }
 }
.......

### Related issues

there is not another issue associated with this?

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Hisoka-X** [comment]
This function is very similar to Hocon's Substitutions. You can refer to SeaTunnel's existing Config library to see if it can be implemented directly.
